docker/Dockerfile-pgtests

```diff
@@ -1,12 +1,21 @@
 # Use the Sytest image that comes with a lot of the build dependencies
 # pre-installed
-FROM matrixdotorg/sytest:latest
+FROM matrixdotorg/sytest:bionic
```

Comment:
I suppose we want this old image so we can give it a fair test on old Python and old Postgres versions?

Reply:
That was my thinking---to be conservative. I'm guessing that most newcomers are just going to test against the system python+postgres, and so it's probably easier for them to use the newer ones rather than the older.

(Bionic may be old, but [its EOL is April 2023](https://wiki.ubuntu.com/Releases))